### PR TITLE
Convert build to use shadow-cljs

### LINKED ISSUE #19: convert crypto lib to shadow-cljs

### PR DESCRIPTION
**Pull request by dpetran**
This standardizes the testing across our repos and provides some handy build reports for measuring artifact size.

Fixes #19

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,20 +13,19 @@ package-lock.json: package.json
 
 node_modules: package.json package-lock.json
 
-cljtest:
-	clojure -M:cljtest
+test-clj:
```
Comment (zonotope) [inline Makefile]: Looks like this breaks symmetry with flure.db:

```
[db] [main $]
% >>> make cljtest                                           zonotope@gertrude
clojure -M:cljtest
Checking out: https://github.com/fluree/json-ld.git at 2e69362f611def8cc6793c6ddf24e4671fb6f397
...

------------------------------------------------------
[db] [main $]
% >>> make test-clj                                          zonotope@gertrude
make: *** No rule to make target 'test-clj'.  Stop.
```

Comment (cap10morgan) [reply Makefile]: ...which I'm personally fine with if we like this name better, but we should change db's names too to match. And then maybe leave targets in there for now w/ the old names that just echo e.g. "You want `make test-whatever now`" to nag me into updating my muscle memory. :)

Comment (dpetran) [reply Makefile]: I was following the convention in the json-ld library, so it looks like the harmonizing needs to go a bit further. I guess I like the <action>-<target> formulation but it's pretty arbitrary.

Comment (dpetran) [reply Makefile]: Alright, standardized on cljtest and cljstest :+1:

**Review by cap10morgan — APPROVED** [review]
🦈

**Pull request merged**